Source/Features/Store/NotificationStorePublisher.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CombineNotificationStore.swift
+//  MagicBell
+//
+//  Created by Joan Martin on 23/12/21.
+//
+
+import Foundation
+import Combine
+
+@available(iOS 13.0, *)
+/// Combine Publisher of the different attributes a NotificationStore has.
+/// Access a publisher via a notification store: `store.publisher()`
+public class NotificationStorePublisher: NotificationStoreCountObserver, NotificationStoreContentObserver {
```

Comment:
@jfilowk rename to `NotificationStorePuiblishers` and update class comments documentation accordingly with the public access to the instance from the store.